.github/workflows/bindings_python.yml

```diff
@@ -57,8 +57,8 @@ jobs:
         working-directory: "bindings/python"
         run: |
           set -e
-          python -m pip install dist/opendal-*.whl
-          python -m pip install -r test_requirements.txt
+          python -m pip install -e .[test]
+          python -m pip install dist/opendal-*.whl --force-reinstall
```

Comment:
A temporary solution that will be more convenient once https://github.com/pypa/pip/issues/8049 is resolved.

Reply:
Um, what are the benefits of this PR? I would rather close it.

Reply:
I'm not too familiar with maturin yet, maybe there is a better way. Give me a few minutes.

Reply:
Let's seek some help from @messense

Reply:
You can simply remove `python -m pip install dist/opendal-*.whl --force-reinstall` and the `- uses: PyO3/maturin-action@v1` step above because `pip install -e .[test]` also builds with maturin internally.

Reply:
If `pip install opendal.whl` is not a necessary step, we can change it to

```
maturin develop -E test --release
pytest
```

Reply:
Be warned that `maturin develop` requires a virtualenv.

Reply:
thanks!